build.rs

```diff
@@ -230,13 +230,29 @@ fn parse_header_defines(header_path: impl AsRef<Path>) -> Result<HashMap<String,
     Ok(definitions)
 }
 
-fn fix_config_map(mut config_map: HashMap<String, String>) -> HashMap<String, String> {
+fn fix_config_map(
+    mut config_map: HashMap<String, String>,
+    interpreter_config: &InterpreterConfig,
+) -> HashMap<String, String> {
+    if interpreter_config.version.major == 3 && interpreter_config.version.minor.unwrap_or(0) > 7 {
+        if let Some("1") = config_map.get("Py_DEBUG").as_ref().map(|s| s.as_str()) {
+            config_map.insert("Py_REF_DEBUG".to_owned(), "1".to_owned());
+        }
+        if let Some("1") = config_map.get("Py_TRACE_REFS").as_ref().map(|s| s.as_str()) {
+            config_map.insert("Py_TRACE_REFS".to_owned(), "1".to_owned());
+        }
```

Comment:
This code looks like a no-op, please delete? It just sets `Py_TRACE_REFS` to `1` if `Py_TRACE_REFS` == `1`.

```suggestion
```

Reply:
Oh, you are right. get config can get this info. I will fix.